oidc-client-app-samples/android-client-app-sample/app/src/main/java/com/wso2is/androidsample/utils/Constants.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2is.androidsample.utils;
+
+/**
+ * Stores the constants.
+ */
+public final class Constants {
+
+    // constants in UserInfoRequest class
+    public static final String AUTHORIZATION = "Authorization";
+    public static final String BEARER = "Bearer ";
+    public static final String UTF_8 = "UTF-8";
+
+    // constants in ConfigManager class
+    public static final String PREFS_NAME = "config";
+    public static final String KEY_LAST_HASH = "lastHash";
+    public static final String HTTP = "http";
+    public static final String HTTPS = "https";
+
+    // constants in AuthStateManager class
+    public static final String STORE_NAME = "AuthState";
+    public static final String KEY_STATE = "state";
+
+    // constants in ConnectionBuilderForTesting class
+    public static final String SSL = "SSL";
```

Comment:
Better not use SSL, but TLS, specifically TLSv1.2

Reply:
Okay, changed SSL to TLS.